core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -0,0 +1,1121 @@
+fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Global): Color {
+    return when (token) {
+        OudsColorKeyToken.Global.TransparentDefault -> globalColors.transparentDefault
+    }
+}
+
+val OudsColorSemanticTokens.materialLightColorScheme: ColorScheme
+    get() = lightColorScheme(
+        primary = backgroundColorTokens.bgBrandPrimaryLight,
+        onPrimary = contentColorTokens.contentDefaultLight,
+        primaryContainer = backgroundColorTokens.bgPrimaryLight,
+        onPrimaryContainer = contentColorTokens.contentDefaultLight,
+        inversePrimary = backgroundColorTokens.bgBrandPrimaryDark,
+        secondary = backgroundColorTokens.bgSecondaryLight,
+        onSecondary = elevationColorTokens.elevationDragOnBgSecondaryLight,
+        secondaryContainer = backgroundColorTokens.bgEmphasizedLight,
+        onSecondaryContainer = contentColorTokens.contentDefaultDark,
+        tertiary = backgroundColorTokens.bgTertiaryLight,
+        onTertiary = contentColorTokens.contentDefaultDark,
+        tertiaryContainer = decorativeColorTokens.decorativeBrandTertiaryLight,
+        onTertiaryContainer = contentColorTokens.contentDefaultOnBgEmphasizedLight,
+        background = backgroundColorTokens.bgPrimaryLight,
+        onBackground = contentColorTokens.contentDefaultLight,
+        surface = elevationColorTokens.elevationRaisedLight,
+        onSurface = contentColorTokens.contentDefaultLight,
+        surfaceVariant = elevationColorTokens.elevationModalLight,
+        onSurfaceVariant = contentColorTokens.contentDefaultLight,
+        surfaceTint = elevationColorTokens.elevationModalLight,
+        inverseSurface = elevationColorTokens.elevationRaisedDark,
+        inverseOnSurface = contentColorTokens.contentDefaultDark,
+        error = backgroundColorTokens.bgStatusNegativeMutedLight,
+        onError = contentColorTokens.contentOnActionNegativeLight,
+        errorContainer = backgroundColorTokens.bgStatusNegativeEmphasizedLight,
+        onErrorContainer = contentColorTokens.contentOnActionDisabledOnBgEmphasizedLight,
+        outline = borderColorTokens.borderBrandPrimaryLight,
+        outlineVariant = borderColorTokens.borderBrandPrimaryLight,
+        scrim = contentColorTokens.contentBrandPrimaryLight,
+    )
+
+val OudsColorSemanticTokens.materialDarkColorScheme: ColorScheme
+    get() = darkColorScheme(
+        primary = backgroundColorTokens.bgBrandPrimaryDark,
+        onPrimary = contentColorTokens.contentDefaultDark,
+        primaryContainer = backgroundColorTokens.bgPrimaryDark,
+        onPrimaryContainer = contentColorTokens.contentDefaultDark,
+        inversePrimary = backgroundColorTokens.bgBrandPrimaryLight,
+        secondary = backgroundColorTokens.bgSecondaryDark,
+        onSecondary = elevationColorTokens.elevationDragOnBgSecondaryDark,
+        secondaryContainer = backgroundColorTokens.bgEmphasizedDark,
+        onSecondaryContainer = contentColorTokens.contentDefaultOnBgEmphasizedDark,
```

Comment:
Light value for `onSecondaryContainer` is `contentDefaultDark`, should we use `contentDefaultLight` here?

Same question for `onErrorContainer` where `contentOnStatusNegativeEmphasizedDark` is used although the light version is `contentOnActionDisabledOnBgEmphasizedLight`.